pytest_pgsql/tests/conftest.py

```diff
@@ -35,10 +35,10 @@ def clean_pgdb(postgresql_db):  # pragma: no cover
 
 
 @pytest.fixture(params=['non-transacted', 'transacted'])
-def clean_db(database_uri, request):
+def clean_db(database_uri, transacted_postgresql_db, postgresql_db, request):
```

Comment:
`transacted_postgresql_db, postgresql_db` are fixtures? Will importing both of them cause both of them to run, even if you only use one (not sure if running means starting a DB instance or opening a connection to it).

Reply:
Yeah they are / it will. Not ideal. I was trying to avoid this by refactoring the `create_fixture` method on those 2 classes but it was a struggle.